algorithm/kapacity/timeseries/forecasting/train.py

```diff
@@ -0,0 +1,118 @@
+#  Copyright 2023 The Kapacity Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+import argparse
+import yaml
+import pandas as pd
+
+import kapacity.metric.query as query
+import kapacity.timeseries.forecasting.forecaster as forecaster
+
+
+def main():
+    # 1. parse arguments
+    args = parse_args()
+
+    # 2. load training config
+    config = load_config(args.config_file)
+
+    # 3. fetch training dataset
+    df = fetch_train_data(args, config)
+
+    # 4. train the model
+    train(args, config, df)
+
+    return
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='args of timeseries forcasting training')
+    parser.add_argument('--dataset-file', help='path of training dataset file, if set, will load dataset from this '
+                                               'file instead of fetching from metrics server', required=False)
+    parser.add_argument('--config-file', help='path of training config file', required=True)
+    parser.add_argument('--metrics-server-addr', help='address of the gRPC metrics provider server', required=False)
+    parser.add_argument('--model-save-path', help='dir path where the model and its related files would be saved in',
+                        required=True)
+
```

Comment:
add a new arg:
`parser.add_argument('--dataloader-num-workers', help='number of worker subprocesses to use for data loading', required=False, default=0`)